src/App.tsx

```diff
@@ -141,10 +149,20 @@ export const App: React.FC = () => {
 
       {/* Notification is shown in case of any error */}
       {/* Add the 'hidden' class to hide the message smoothly */}
-      <div className="notification is-danger is-light has-text-weight-normal">
-        <button type="button" className="delete" />
+      <div
+        className={error
+          ? 'notification is-danger is-light has-text-weight-normal'
+          : 'notification is-danger is-light has-text-weight-normal hidden'}
+      >
+        <button
+          type="button"
+          className="delete"
+          onClick={() => setError(false)}
+        />
 
         {/* show only one message at a time */}
+        Unable to load a todo
```

Comment:
```suggestion
        Unable to load a todo
        <br />
        Unable to add a todo
        <br />
        Unable to delete a todo
```

think now this lines can be replaced just by you errorMessage state, you will get right message based on error from server, e.g `{errorMessage}`